enterprise/javaee.wildfly/src/org/netbeans/modules/javaee/wildfly/ide/ui/WildflyPluginUtils.java

```diff
@@ -440,6 +442,10 @@ private static Attributes findManifestAttributes(File serverPath, String slot) {
                     JarFileSystem featurePackProductConfJar = new JarFileSystem(file);
                     return featurePackProductConfJar.getManifest().getMainAttributes();
                 }
+                if (file.getName().startsWith("wildfly-ee-feature-pack-product-conf") && file.getName().endsWith(".jar")) {
+                    JarFileSystem featurePackProductConfJar = new JarFileSystem(file);
+                    return featurePackProductConfJar.getManifest().getMainAttributes();
+                }
```

Comment:
this looks like it would resolve the wildfly issue @mrcasio mentioned https://github.com/apache/netbeans/pull/7055#issuecomment-2361272391

Reply:
i can't test JBoss EAP 8 since it requires an account to download, I tested simple start/stop using WildFly 30, 31, 32 and 33 and it worked. However, it did also work on NetBeans 23 already - so I can't reproduce the wildfly issue.

Reply:
> i can't test JBoss EAP 8 since it requires an account to download, I tested simple start/stop using WildFly 30, 31, 32 and 33 and it worked. However, it did also work on NetBeans 23 already - so I can't reproduce the original wildfly issue.

It doesn't appear that you need an account to download JBoss 8.0:  https://developers.redhat.com/products/eap/download
Let me know if there is anything else you need to merge this pull request.

Reply:
@HowardUngar the list is public, the download is not.